docs/fidesops/docs/postman/Fidesops.postman_collection.json

```diff
@@ -4753,6 +4757,114 @@
 					"response": []
 				}
 			]
+		},
+		{
+			"name": "Consent Request",
+			"item": [
+				{
+					"name": "Create Verification Code for Consent Request",
+					"request": {
+						"method": "POST",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n  \"phone_number\": \"{{phone_number}}\",\n  \"email\": \"{{email}}\"\n}",
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "{{host}}/consent-request",
+							"host": [
+								"{{host}}"
+							],
+							"path": [
+								"consent-request"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Verify Code and Return Current Preferences",
+					"request": {
+						"method": "POST",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n  \"code\": \"{{verification_code}}\"\n}",
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "{{host}}/consent-request/{{consent_request_id}}/verify",
+							"host": [
+								"{{host}}"
+							],
+							"path": [
+								"{{consent_request_id}}",
+								"consent-request"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Verify Code and Save Preferences",
+					"request": {
+						"method": "PATCH",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n  \"identity\": {\n    \"phone_number\": \"{{phone_number}}\",\n    \"email\": \"{{email}}\"\n  },\n  \"consent\": [\n    {\n      \"data_use\": \"{{data_use}}\",\n      \"data_use_description\": \"{{data_use_description}}\",\n      \"opt_in\": {{opt_in}}\n    }\n  ],\n  \"code\": \"string\"\n}",
```

Comment:
You might map code to `"{{verification_code}}"` instead of "string" in this request body as you do in the verify code and get preferences endpoint so it's clear this needs to be replaced.

I'd also add `verification_code` as a variable, I missed that this one isn't there.